pygmt/helpers/decorators.py

```diff
@@ -15,13 +15,14 @@
 COMMON_OPTIONS = {
     "R": """\
         region : str or list
-            *Required if this is the first plot command*.
-            ``'xmin/xmax/ymin/ymax[+r][+uunit]'``.
-            Specify the region of interest.""",
+            *xmin/xmax/ymin/ymax*\ [**+r**\ ][**+u**\ *unit*].
+            Specify the region of interest. This is a required argument if this
+            is the first plot command.""",
     "J": """\
```

Comment:
```suggestion
    "J": r"""\
```
Same here, use r-string.